src/handbook/customer/customer-success.md

```diff
@@ -174,3 +174,8 @@ and getting more value from the product
 1. Data collection -- Provide space and time for the customer to provide feedback
 on the product and experience. Allow them to request feature ideas, or ask for bug
 fixes.
+
+## Churned customers
+
+When an annually-billed customer chooses not to renew their subscription, the assigned Account Executive should login into Stripe and cancel the subscription. This will effectively close the account on the platform and prevent invoices to continue being sent.
```

Comment:
@joepavitt @knolleary Can we verify this would work for FFC too? Self-hosted this process is fine.

Reply:
Annually billed customers on FFC are in manually billing mode. This means there is no tie between stripe and the platform - so cancelling the Stripe subscription will have no effect on the platform - the team will remain active.

Reply:
@knolleary Can you propose a way to ensure the team is no longer active?

Reply:
Have raised #2527 to track and update the docs